site/docs/account/applications.mdx

```diff
@@ -36,7 +36,7 @@ Applications are where you configure webhook URLs for Bandwidth to send either m
 
 Each application can be assigned to as many Locations (Sip-Peers) as needed.  However, each **Location** can only be associated with a single **Application** per service type (Voice or Messaging).
 
-The remainder of this guide covers how to create applications via the API. For a detailed guide using the UI, please see our [account setup guide](./setup).
+The remainder of this guide covers how to create applications via the API. For a detailed guide using the UI, please see our [account setup guide](/docs/account/setup).
```

Comment:
```suggestion
The remainder of this guide covers how to create applications via the API. For a detailed guide to using the UI, please see our [account setup guide](/docs/account/setup).
```
Your suggestion was fine too; looks like it just didn't make it in.

Reply:
Whoops! I blame github hiding conversations from me...